src/environmentd/tests/util.rs

```diff
@@ -277,6 +285,33 @@ pub fn start_server(config: Config) -> Result<Server, anyhow::Error> {
     let postgres_factory = StashFactory::new(&metrics_registry);
     let secrets_controller = Arc::clone(&orchestrator);
     let connection_context = ConnectionContext::for_tests(orchestrator.reader());
+    let (tracing_handle, tracing_guard) = if config.enable_tracing {
+        let config = TracingConfig::<fn(&tracing::Metadata) -> sentry_tracing::EventFilter> {
+            service_name: "environmentd",
+            stderr_log: StderrLogConfig {
+                format: StderrLogFormat::Json,
+                filter: Targets::default(),
+            },
+            opentelemetry: Some(OpenTelemetryConfig {
```

Comment:
Can this be `None`?

Reply:
Are you referring to the `opentelemetry` field? That cannot be `None` because otherwise we wouldn't setup the `opentelemetry` [tracing layer](https://github.com/MaterializeInc/materialize/blob/main/src/ore/src/tracing.rs#L265) which is necessary for the test